sites/sample.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+$obj = new clazz();
+
+echo "Its ok";
+
+print_r($_SERVER['nokey']);
+
+
+// daskhdkhsadkhakdhakhsdkahsdkhasdjaldmwqldmlmdlmasldmsalmdalsmdlaksmdlsamdlamsdlmasldmaldmalsmdlasmdlsamdlamsdlamsdlmasldmasldmalsdmalsmdlasmdlasmdlasmdlmasldmaldmad
+
+
+$have_time = true;
+$have_money = false;
+$take_vacation = $have_time and $have_money;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace "and" with "&&". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS2010)